src/App.jsx

```diff
@@ -2,86 +2,54 @@ import React, { useState } from "react";
-    { value: "5", type: "Five" },
-    { value: "6", type: "Six" },
-    { value: "7", type: "Seven" },
-    { value: "8", type: "Eight" },
-    { value: "9", type: "Nine" },
-    { value: "10", type: "Ten" },
-    { value: "J", type: "Jack" },
-    { value: "Q", type: "Queen" },
-    { value: "K", type: "King" },
-    { value: "A", type: "Ace" },
-    { value: "JK", type: "Joker" },
-    { value: "Queen", type: "Queen" },
-  ];
-
+  const [color, setColor]=useState('')
   const validateCard = () => {
     if (cardValue && suit) {
-      const foundCard = cardData.find((card) => card.value === cardValue);
-      
-      if (foundCard) {
-        if(foundCard.value==="JK"){
-          setSelectedCard(foundCard);
-          if(suit ==="Hearts"|| suit==="Diamonds" ){
-            alert(`This is a ${foundCard.type} of red`)
-          }
-          else{
-          alert(`This is a ${foundCard.type} of black`)}
-        }
-        else{
-        setSelectedCard(foundCard);
-        alert(`This is a ${foundCard.type} of ${suit}`);
-      } }
-      else {
-        alert("Card not found.");
+      const cardType = cardValues[cardValue];
+      const cardColor = colors[color]
+      if (cardType && cardColor) {
+        alert(`${cardType} of ${suits[suit]} ${cardColor}`);
+      }
+      else{
+        alert(`incorect card`)
       }
-    } else {
-      alert("Please enter card value and select suit.");
     }
   };
- 
-  const renderCardMessage = () => {
-    if (selectedCard) {
-      return <p>This is a {selectedCard.type} of {suit}</p>;
-    }
-    return null;
+  const handleSuitChange = (e) => {
+    setSuit(e.target.value);
+    setColor(e.target.value);
   };
 
   return (
     <>
-    <div className="wrapper">
-    <ImSpades className="spades" />
-   
-    <div className="validator">
-    <h1>Input your card</h1>
-      <input
-        type="text"
-        placeholder="Enter card value"
-        value={cardValue}
-        onChange={(e) => setCardValue(e.target.value)}
-      />
-      <select value={suit} onChange={(e) => setSuit(e.target.value)}>
-        <option value="">Select suit</option>
-        <option value="Hearts">♥</option>
-        <option value="Diamonds">♦</option>
-        <option value="Clubs">♣</option>
-        <option value="Spades">♠</option>
-      </select>
-      <button onClick={validateCard}>Validate</button>
-      
-    </div>
-    <PiHeartStraightFill className="hearts" />
-    </div>
+      <div className="wrapper">
+        <ImSpades className="spades" />
+        <div className="validator">
+          <h1>Input your card</h1>
+          <input
+            type="text"
+            placeholder="Enter card value"
+            value={cardValue}
+            onChange={(e) => setCardValue(e.target.value)}
+          />
+          <select value={suit} onChange={handleSuitChange}>
+            <option value="">Select suit</option>
+            {Object.entries(suits).map(([key, value]) => (
```

Comment:
Should we use Object.entries here?

Reply:
we can use Object.keys but then we can't see the suit names